### PR TITLE
chore: Merge master branch into feature branch

### PR DESCRIPTION
**Pull request by hengkx**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [x] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/popconfirm/index.en-US.md](https://github.com/ant-design/ant-design/blob/mergefeature/components/popconfirm/index.en-US.md)
[View rendered components/popconfirm/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/mergefeature/components/popconfirm/index.zh-CN.md)
[View rendered docs/react/contributing.zh-CN.md](https://github.com/ant-design/ant-design/blob/mergefeature/docs/react/contributing.zh-CN.md)
[View rendered docs/react/migration-v4.en-US.md](https://github.com/ant-design/ant-design/blob/mergefeature/docs/react/migration-v4.en-US.md)
[View rendered docs/spec/introduce.en-US.md](https://github.com/ant-design/ant-design/blob/mergefeature/docs/spec/introduce.en-US.md)

### CODE REVIEW

**Comment by hengkx** [comment]
`License Compliance` 为什么一直 都处于 `Pending`

File: components/rate/index.tsx [hunk 1]
```diff
@@ -26,50 +27,36 @@ interface RateNodeProps {
...
-    const rateProps = omit(restProps, ['tooltips']);
-    const ratePrefixCls = getPrefixCls('rate', prefixCls);
-
-    return (
-      <RcRate
-        ref={this.saveRate}
-        characterRender={this.characterRender}
-        {...rateProps}
-        prefixCls={ratePrefixCls}
-        direction={direction}
```
Comment (afc163) [inline components/rate/index.tsx:67]: direction 改漏了。

Comment (afc163) [reply components/rate/index.tsx:67]: 对比一下 https://github.com/ant-design/ant-design/pull/23321/files 的 diff。

Comment (hengkx) [reply components/rate/index.tsx:67]: 重新来一个

**Pull request closed without merging**